python/feathub/feature_views/sql_feature_view.py

```diff
@@ -0,0 +1,137 @@
+# Copyright 2022 The Feathub Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Dict, Sequence, Optional, List
+
+from feathub.common.exceptions import FeathubException
+from feathub.feature_views.feature import Feature
+from feathub.feature_views.feature_view import FeatureView
+from feathub.registries.registry import Registry
+from feathub.table.schema import Schema
+from feathub.table.table_descriptor import TableDescriptor
+
+
+class SqlFeatureView(FeatureView):
+    """
+    Acquires features by evaluating the given SQL query statement using a specific
```

Comment:
Can you also document this feature view in https://github.com/alibaba/feathub/blob/master/docs/architecture.md?